### PR TITLE
Add a script to validate dashboards.

### PR DESCRIPTION
**Pull request by fcuny**
This script validate that all the categories contains a description (the `dash.yaml` file), and that all the `.graph` files can be parsed with `GraphiteGraph`.

When a new graph is created, if there's a syntax error in the graph description, the application will be unable to display it.  If the dashboards are stored in some kind of repository, before pulling them to the server, or before pushing them, this script can be run to validate that the graphs can be loaded by `GraphiteGraph` and that all categories contains the dash description (e.g: a git hook, Jenkin's job, etc.)

### CODE REVIEW

File: tools/dashboards-validation.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env ruby
+$LOAD_PATH << './lib'
```
Comment (ripienaar) [inline tools/dashboards-validation.rb]: this is probably not needed in the final shipped one?

**Comment by ripienaar** [comment]
very kewl!

File: tools/dashboards-validation.rb (hunk 1 above)
Comment (fcuny) [inline tools/dashboards-validation.rb]: yes, this line can be deleted in the shipped version.  If you're fine with this patch I'll delete it now.

Comment (ripienaar) [inline tools/dashboards-validation.rb]: yeah looks sweet, kill that and I'll merge :)

Comment (fcuny) [inline tools/dashboards-validation.rb]: Done  :)

Thanks.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**